### ISSUE TITLE
Start training from a custom pretrained model

### ISSUE THREAD

**Issue by AlexPasqua** [ultralytics/ultralytics#19296]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Hello, I want to train a YOLO model starting from a specific pretrained model that I have on my machine.
I specify this model using the `model` parameter of the class `YOLO`: `my_model = YOLO(model="path/to/my/model.pt")`, but I notice that at training start, yolov11n.pt gets downloaded, which is the default model.

Why is yolov11.pt getting downloaded if I'm specifying my model?
How can I be sure that it's using my pretrained model to start with?

Thanks for any clarification

**Comment by UltralyticsAssistant** [comment]
👋 Hello @AlexPasqua, thank you for your interest in Ultralytics 🚀! We recommend checking out our [Docs](https://docs.ultralytics.com/) for guidance. They include resources for both [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples, which might help clarify your question.

To address your query about custom pretrained models, if this is a 🐛 **Bug Report**, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) that demonstrates the issue, along with details about your environment and your `model` parameter setup. This will help us debug further.

On the other hand, if this is a ❓ **Question** related to custom training, ensure to provide as much detail as possible about the setup, including:
- The path to your custom model
- Any related training scripts or logs you’re using
- Verification steps you've taken, such as checking your model’s initialization

To ensure you're working with the latest features and fixes, upgrade to the latest Ultralytics version with the following command:

```bash
pip install -U ultralytics
```

## Precompiled Environments

You can test and verify your setup using any of the following precompiled environments, which come with YOLO fully pre-installed:
- **Free GPU environments** (like Colab or Kaggle):  
  <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a>  
  <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a>  
  <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Other verified environments:**  
  [Google Cloud Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/) | [AWS Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/) | [Docker](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/)  
  <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

If you'd like quick feedback or discussion from the Ultralytics community, join us on [Discord](https://discord.com/invite/ultralytics) 🎧 or [Discourse](https://community.ultralytics.com/). You can also share your solutions or questions on our [Subreddit](https://reddit.com/r/Ultralytics).

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>  
This badge indicates whether all Ultralytics CI tests are currently passing on macOS, Windows, and Ubuntu environments.

Please note that this is an automated response 🤖, but don’t worry—an Ultralytics engineer will review this issue and respond as soon as possible. Thank you for your patience! 😊

**Comment by Y-T-G** [comment]
Duplicate of https://github.com/ultralytics/ultralytics/issues/17011

**Comment by glenn-jocher** [comment]
@AlexPasqua thank you for your understanding! We're glad the previous response helped. For any future questions, remember to check our comprehensive documentation at https://docs.ultralytics.com. The YOLO community and Ultralytics team are always here to help with your computer vision projects. 🚀
